content/directory/PLU Design.md

```diff
@@ -3,7 +3,7 @@ title: "PLU Design"
 cover: "./content/blog/imgs/hamburgers.png"
 website: "https://www.plu.edu/artd/"
```

Comment:
Same as previous. Schools should probably only have one top level entry, even though they may have multiple programs.